provider/source/data/debug/duration/digital@1/en.json

```diff
@@ -0,0 +1,16 @@
+{
+  "separator": ":",
+  "hms_padding": {
+    "h": 1,
+    "m": 1,
+    "s": 1
+  },
+  "hm_padding": {
+    "h": 1,
+    "m": 1
+  },
+  "ms_padding": {
+    "m": 1,
+    "s": 1
+  }
+}
```

Comment:
this can be optimized, but we can do it in another PR: @kartva , just file a but to optimize this.

Reply:
To clarify, you are recommending that I file an issue to optimize this?

Reply:
I think @younies intended to say "file a bug", so yes. Alternatively, if you already have an issue where you are tracking follow-ups, you can add it there.

Reply:
This should be easy to bitpack: I think the only possible values are

```rust
pub enum HmsPadding {
    None,
    Hour,
    Minute,
    Second,
    HourMinute,
    HourSecond,
    HourMinuteSecond,
}
```

Are there any locales with patterns such as the following? If not, we can cut down the options further:

- `h:m:ss`
- `h:mm:s`
- `h:m:s`
- `m:s`